nowcasting_dataset/data_sources/fake.py

```diff
@@ -172,32 +173,65 @@ def topographic_fake(batch_size, image_size_pixels):
     return Topographic(xr_dataset)
 
 
+def add_uk_centroid_osgb(x, y):
```

Comment:
I'm a bit lost in this function.

What does x and y do to the random lat/lon values?

Is it more like "create a random OSGB value in the centre of UK influenced by x,y <magic>"?
I think it would help to change the comment and make the function name a bit more descriptive.